### PR TITLE
Add the Sign-in logic

### PR DESCRIPTION
**Pull request by OlexG**

### CODE REVIEW

**Comment by OlexG** [comment]
Closes #16 and #15

**Review by reakunen — APPROVED** [review]
very good job

**Comment by shua1090** [comment]
image

I like the popout, but the white font on logout is kinda mid ngl:
image

File: pages/signin.tsx [hunk 1]
```diff
@@ -2,36 +2,73 @@ import "../app/globals.css";
...
+            <GoogleLogin
+              onSuccess={handleLogin}
+              onError={() => {
+                console.log("Login Failed");
+              }}
+            />
+          </GoogleOAuthProvider>
+        </div>
+      ) : (
+        <div className="w-full flex flex-col items-center justify-center">
```
Comment (shua1090) [inline pages/signin.tsx:57]: image
Just fix the font and I'll approve

**Review by shua1090 — CHANGES_REQUESTED** [review]

**Review by jeremiahliao — APPROVED** [review]

**Review by shua1090 — APPROVED** [review]
good stuff!

**Pull request merged**